src/main/java/Duke.java

```diff
@@ -1,10 +1,175 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner scanner = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>(100);
+        String message = "____________________________________________________________\n" +
+                "Hello! I'm Jux\n" +
+                "What can I do for you?\n";
+        String end = "____________________________________________________________\n" +
+                "Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________"
+                ;
+        FileManipulation dataFile = new FileManipulation();
+        if (dataFile.getDoesExist()) {
+            try {
+                dataFile.loadFile(list);
+                for (int i = 0; i < list.size();i++) {
+                    System.out.println(list.get(i));
+                }
+
+            } catch(FileNotFoundException e) {
+                System.out.println("File not found");
+            }
+        } else {
+            System.out.println(message);
+        }
+
+        while(true) {
+
+            try {
+                String input = scanner.nextLine();
+                if (input.equals("bye")) {
+                    break;
+                } else if(input.equals("list")) {
```

Comment:
Maybe should add a blank space after "if"?
```suggestion
                } else if (input.equals("list")) {
```